docs/segments/manage_segments.rst

```diff
@@ -3,4 +3,493 @@
+
+* **Less than or equal** - The value on the Contact record is at an earlier date in time than X date. For example, **Less than or equal today** means anytime from the beginning of time until today.
+
+* **Empty** - No value exists in the field on the Contact record.
+
+* **Not empty** - A value exists in the field on the Contact record.
+
+* **Like** - This operator isn't supported in date or date-time fields, and shouldn’t be used.
+
+* **Not like** - This operator isn't supported in date or date-time fields, and shouldn’t be used.
+
+* **Regexp** - Segment includes Contacts with values that match the specified regular expression pattern. If you aren't proficient with regular expression, don’t use this operator.
+
+* **Not regexp** - Segment includes Contacts with values that don’t match the specified regular expression pattern. If you aren't proficient with regular expressions, don’t use this operator.
+
+* **Starts with** - Segment includes Contacts whose field values begin with the specified numbers. These filter values should generally reference years (or years and months).
+
+For example, A value of 19 matches any Contacts whose field value has a year in the 1900&nbsp;s. A value of 200 matches Contacts with a year value between 2000 and 2009 and a value of 2020-11 matches Contacts with a field value in November 2020.
+
+* **Ends with** - Segment includes Contacts whose field values end with the specified numbers. These filter values should generally reference days (or months and days).
+
+For example, a value of 1 matches anyone whose field value is on the 1&nbsp;st, 21&nbsp;st, or 31&nbsp;st of any month but a value of 01 matches the 1&nbsp;st of a month. A value of 01-01 finds Contacts whose value is for January 1 of any year.
+
+* **Contains** - Segment includes Contacts with the specified filter value anywhere in the field value. 
+
+.. image:: images/operators-2.png
+    :alt: Screenshot showing Operators.
+
+Once you have selected the field you can then choose the type of operation to perform. These vary depending on the way you wish to filter your Contacts.
+
+.. vale off 
+
+Matching part of a string
+=========================
+
+.. vale on
+
+There are 5 filters you can use for matching part of a string - ``starts with``, ``ends with``, ``contains``, ``like`` and ``regexp``.
+First three filters match strings as you enter it. ``like`` filter is for advanced Users - you can specify which type you want to use with ``%`` character:
+
+* ``My string%`` is the same as ``starts with`` filter with ``My string`` value.
+
+* ``%My string`` is the same as ``ends with`` filter with ``My string`` value.
+
+* ``%My string%`` is the same as ``contains`` filter with ``My string`` value.
+
+* ``My string`` is the same as ``contains`` filter with ``My string`` value.
+
+A few notes for text filters:
+
+* ``starts with``, ``ends with``, ``contains`` filters were added later than the ``like`` one, so you can easily specify what you need now.
```

Comment:
```suggestion
* ``starts with``, ``ends with``, ``contains`` filters should be used rather than ``like`` as they are more specific, and therefore can be more performant.
```